pyanaconda/anaconda_log.py

```diff
@@ -32,6 +32,10 @@
 STDOUT_FORMAT = "%(asctime)s %(message)s"
 DATE_FORMAT = "%H:%M:%S"
 
+# the Anaconda log uses structured logging
+ANACONDA_ENTRY_FORMAT = "%(asctime)s,%(msecs)03d %(levelname)s %(log_prefix)s: %(message)s"
```

Comment:
I think that we should at least investigate the second option, but we can do that in a different issue :-)